### PR TITLE
Add house

### LINKED ISSUE #5: [1.5pts] Create House form modal
Create a page that has a form that allows new Dress to be added.

### LINKED ISSUE #8: [2pts] Add House link to navigation
Create a link of add Houses page to navigation

### PR DESCRIPTION
**Pull request by brhanuhailu**
# In this branch I:
- [ ] Add House 
- [ ] Add little css for List of House in Home page
- [ ] But not add css for Add - house page(only class and id)
- [ ] Add a link of add Houses page to navigation
image

Result in home page displayed added house
image

image

### CODE REVIEW

**Comment by brhanuhailu** [comment]
Hello Team, I added the two cards since the one is just creating a link they are related to. but I have modified the House components. but I will do it soon b/c it gives me more assignment to add-courses.

File: src/components/AddHouse.jsx
Comment (yashodhicy) [inline src/components/AddHouse.jsx:1]: you have did a nice job here . working perfect. 👍🏻 🍰 🥳 
image

File: src/components/componentsCss/houses.css
Comment (yashodhicy) [inline src/components/componentsCss/houses.css:1]: Nice UI 👏🏻 
image

**Review by yashodhicy — CHANGES_REQUESTED** [review]
# Changes Required
You have done a great job here but house details page is not rendering details
image

image

**Review by yashodhicy — APPROVED** [review]
# Approved

You have done a great job  here. Your project is approved. 💛 ㊗️

**Pull request merged**